vars/perfRegressionParallelPipeline.groovy

```diff
@@ -74,129 +111,44 @@ def call(Map pipelineParams) {
                             } else {
                                 perf_test = "${pipelineParams.test_name}.${sub_test}"
                             }
-
+                            test_params["${sub_test}"] = params + ["test_name":"${perf_test}", "test_id":UUID.randomUUID().toString()]
+                            print("test_params=$test_params")
+                            def completed_stages = [:]
+                            completed_stages["${perf_test}"] = [:]
                             tasks["sub_test=${sub_test}"] = {
-                                node(builder.label) {
```

Comment:
why are you removing this, it's an important part so it can be split across multiple workers...